pkgs/development/tools/haskell/ihaskell/wrapper.nix

```diff
@@ -9,7 +9,7 @@ let
   ihaskellSh = writeScriptBin "ihaskell-notebook" ''
     #! ${stdenv.shell}
     export GHC_PACKAGE_PATH="$(echo ${ihaskellEnv}/lib/*/package.conf.d| tr ' ' ':'):$GHC_PACKAGE_PATH"
-    export PATH="${stdenv.lib.makeBinPath ([ ihaskellEnv jupyter ])}\${PATH:+':'}$PATH"
+    export PATH="${stdenv.lib.makeBinPath ([ ihaskellEnv jupyter ])}'' + "\${PATH:+':'}" + ''$PATH"
```

Comment:
This doesn't actually work; it results in:

```console
$ NIXPKGS_ALLOW_BROKEN=1 nix-build -A ihaskell
...
/nix/store/bd3xj2p79zrqkz9xr8daq6752avvv90f-stdenv-linux/setup: eval: line 1275: unexpected EOF while looking for matching `"'
```

```suggestion
    export PATH="${stdenv.lib.makeBinPath ([ ihaskellEnv jupyter ])}''${PATH:+':'}$PATH"
```
You can simply use `''${foo}` as an escape, which results in the expected result:
```
# ./result/bin/.ihaskell-notebook-wrapped

#! /nix/store/z4ajipns0l1s8b2lrgpy6nng4cys7h99-bash-4.4-p23/bin/bash
export GHC_PACKAGE_PATH="$(echo /nix/store/h5n6xxgnxchzdxq47ppbq9j2q6nr35xf-ghc-8.8.3-with-packages/lib/*/package.conf.d| tr ' ' ':'):$GHC_PACKAGE_PATH"
export PATH="/nix/store/h5n6xxgnxchzdxq47ppbq9j2q6nr35xf-ghc-8.8.3-with-packages/bin:/nix/store/bpgh69gsklmmcsi0idknn1yihsz9lb45-python3-3.7.6-env/bin${PATH:+':'}$PATH"
/nix/store/h5n6xxgnxchzdxq47ppbq9j2q6nr35xf-ghc-8.8.3-with-packages/bin/ihaskell install -l $(/nix/store/h5n6xxgnxchzdxq47ppbq9j2q6nr35xf-ghc-8.8.3-with-packages/bin/ghc --print-libdir) && /nix/store/bpgh69gsklmmcsi0idknn1yihsz9lb45-python3-3.7.6-env/bin/jupyter notebook
```

Reply:
Thanks for the feedback!  I was unaware of that syntax.  I'll update the commit.